content/influxdb/v2.2/write-data/troubleshoot.md

```diff
@@ -60,25 +77,240 @@ Write requests return the following status codes:
+| `gotType`     | Received [field](/influxdb/cloud/reference/key-concepts/data-elements/#field-value) type: `Boolean`, `Float`, `Integer`, `String`, or `UnsignedInteger`   |
+| `wantType`    | Expected [field](/influxdb/cloud/reference/key-concepts/data-elements/#field-value) type: `Boolean`, `Float`, `Integer`, `String`, or `UnsignedInteger`   |
+| `<timestamp>` | Time the rejected point was logged                                                                                                                        |
+
+#### Find parsing errors
+
+If InfluxDB can't parse a line (e.g. due to syntax problems), the response `message` might not provide details.
+To find parsing error details, query `rejected_points` entries that contain the `error` field (instead of the `count` field).
+
+```js
+from(bucket: "_monitoring")
+    |> range(start: -1h)
+    |> filter(fn: (r) => r._measurement == "rejected_points")
+    |> filter(fn: (r) => r._field == "error")
+```
+
+#### Find data type conflicts and schema rejections
+
+To find `rejected_points` caused by [data type conflicts](#resolve-data-type-conflicts) or [schema rejections](#resolve-explicit-schema-rejections),
+query for the `count` field.
+
+```js
+from(bucket: "_monitoring")
+    |> range(start: -1h)
+    |> filter(fn: (r) => r._measurement == "rejected_points")
+    |> filter(fn: (r) => r._field == "count")
+```
+
+### Resolve data type conflicts
+
+When you write to a bucket that has the `implicit` schema type, InfluxDB compares new points to points that have the same [series](/influxdb/cloud/reference/key-concepts/data-elements/#series).
+If a point has a field with a different data type than the series, InfluxDB rejects the point and logs a `rejected_points` entry.
+The `rejected_points` entry contains one of the following reasons:
+
+| Reason                              | Meaning                                                                                                       |
+|:------                              |:-------                                                                                                       |
+| `type conflict in batch write`      | The **batch** contains another point with the same series, but one of the fields has a different value type.  |
+| `type conflict with existing data`  | The **bucket** contains another point with the same series, but one of the fields has a different value type. |
+
+### Resolve explicit schema rejections
+
+Buckets with the [`explicit` schema type] use [explicit bucket schemas](/influxdb/cloud/organizations/buckets/bucket-schema/).
+When you write to a bucket that uses explicit bucket schemas,
+InfluxDB rejects data that don't conform to one of the configured schemas.
+
+Learn how to interpret `rejected_points` logging for [explicit bucket schemas](/influxdb/cloud/organizations/buckets/bucket-schema/).
```

Comment:
```suggestion
If you write to a bucket with an [explicit schema](/influxdb/cloud/organizations/buckets/bucket-schema/), the data must conform to the schema. Otherwise, InfluxDB rejects the data.

Do the following to interpret explicit schema rejections:
```